spec/inputs/udp_spec.rb

```diff
@@ -1,6 +1,47 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/udp'
+require_relative "../spec_helper"
+require "support/client"
```

Comment:
This need a `require_relative` since the `support` folder is not added to the path it will make this test fail when running the all plugins test.